### PR TITLE
Double Enhance Fixes, fs-fallback, and warning fixes

### LINKED ISSUE #12: double enhance improvements
* [ ] reply to thread, not channel
* [x] add more permissive regex
* [x] adjustable size
* [/] see if sharp has some sharpening features
* [ ] cleanup files post-posting when they are no longer needed

### PR DESCRIPTION
**Pull request by Yhprum**
- fixes log warning for incorrect documentation of util.coffee
- re-adds fs-fallback for emoji cache
- fixes #12 
  - added more regex, size, adjusting, **gif support!!**
  - did not add reply to thread (I like being bombarded with a giant image)
  - not sure what to do about sharpening features

### CODE REVIEW

**Comment by Yhprum** [comment]
Sharp does have the [`.sharpen`](https://sharp.pixelplumbing.com/api-operation#sharpen) method, but I wasn't quite sure how to implement it and I didn't notice much difference for the default settings and a higher sigma value. We could possibly add it in but only if the regex says to add more, like `emojme super enhance to 1024px and sharpen` or have the value of sigma depend on the number of exclamation points you add to `emojme super enhance!!!!11!1` or something dumb like that.

**Comment by Yhprum** [comment]
Now that I looked at it a bit more, for "reply to thread, not channel," was that only in the case where we request the enhance already in a thread or just replying to all requests by starting a thread reply?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "emojme-hubot-plugin",
-  "version": "1.0.1",
+  "version": "1.0.2",
```
Comment (jackellenberger) [inline package.json]: Give yourself at least a minor version bump for this my god!

**Comment by jackellenberger** [comment]
> Sharp does have the [`.sharpen`](https://sharp.pixelplumbing.com/api-operation#sharpen) method, but I wasn't quite sure how to implement it and I didn't notice much difference for the default settings and a higher sigma value. We could possibly add it in but only if the regex says to add more, like `emojme super enhance to 1024px and sharpen` or have the value of sigma depend on the number of exclamation points you add to `emojme super enhance!!!!11!1` or something dumb like that.

we don't have to like the user that much, if someone complains about dog producing fuzzy images they 100% don't know how good they have it.

> Now that I looked at it a bit more, for "reply to thread, not channel," was that only in the case where we request the enhance already in a thread or just replying to all requests by starting a thread reply?

I think the former, but i would imagine when this gets released and everyone goes crazy enhancing images it'll become apparent if it's an issue or not. maybe people want to see 1024px emoji just bombing #chicago with no context :shrugidk:

**Comment by jackellenberger** [comment]
Published! sorry again for being so slow always every time @Yhprum, if you want to email me every day to get on this biz you are free to!

**Pull request merged**